src/main.cc

```diff
@@ -76,8 +76,6 @@
 #include "time/Engine.h"
 #include "tools.h"
 #include "unlinkd.h"
-#include "wccp.h"
-#include "wccp2.h"
```

Comment:
Please make the six affected init/open/close functions private/static. They no longer need to be "published" in header files for reuse outside of their translation units.

Reply:
Thanks. Done in a68fbd1d09